bin/missing-versions

```diff
@@ -7,6 +7,14 @@ PLATFORMS = %w[amd64 arm64].freeze
 # 24.2 supports OpenSSL 3 and modern gcc/autoconf versions
 FIRST_SANE_VERSION = Gem::Version.new("24.2")
 
+def skip_release?(dist:, version:)
+  return true if version == "patch-base-26"
```

Comment:
I was contemplating to add the URL to the release page, but figured it would be in the commit message. I can add something if you'd like

Reply:
I'll close this, we can reopen and make adjustment the next time this happens (if it happens). Feels strange to add the indirection (method) for just one case and it will mess with git blame.